HISTORY.md

```diff
@@ -1,6 +1,21 @@
 # Release Notes
 
-### v1.16.2 - 2024-09-25
+### v1.17.0 - 2024-10-01
+
```

Comment:
```suggestion

This release consolidates the `SingleTableMetadata` and `MultiTableMetadata` classes into one new class simply called `Metadata`. The old classes are now deprecated. The new class has the same structure as the `MultiTableMetadata` class, except it will work with single table synthesizers.

```